src/nls/fi/strings.js

```diff
@@ -624,7 +624,25 @@ define({
     "CMD_TOGGLE_RECENT_PROJECTS"                : "Viimeisimmät projektit",
 
     // extensions/default/WebPlatformDocs
-    "DOCS_MORE_LINK"                            : "Lue lisää"
+    "DOCS_MORE_LINK"                            : "Lue lisää",
+
+    //extensions/default/CodeFolding
+    "COLLAPSE_ALL"                  : "Pienennä kaikki",
```

Comment:
@abose See https://trello.com/c/EwLGRkYe/541-native-submenus.  The only reason we don't have it is that it's a sizeable chunk of work: we'd have to decide on the JS `Menus` API, implement submenus natively on Mac and Win, _and_ implement them HTML for Linux / in-browser before the feature could land in master.